styles.css

```diff
@@ -0,0 +1,186 @@
+* {
+    box-sizing: border-box;
+}
+
+body {
+    margin: 0px;
+    font-family: sans-serif;
+}
+
```

Comment:
Maybe you can categorize the code into "base style" "main style" "grid style" etc to have the code more organized?